DependencyInjection/MonologExtension.php

```diff
@@ -519,12 +519,18 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['level'],
                 $handler['bubble'],
             ]);
-            if ($handler['ident']) {
+
+            if (false === $handler['ident'] || null === $handler['ident']) {
+                $handler['ident'] = 'php';
+                $definition->addArgument($handler['ident']);
+            } elseif ($handler['ident']) {
                 $definition->addArgument($handler['ident']);
             }
```

Comment:
does not work with other "false" values like `0`, `''`, ...
I suggest to always set a value, and use an `if/else` block for that
```suggestion
            if ($handler['ident']) {
                $definition->addArgument($handler['ident']);
            } else {
                $handler['ident'] = 'php';
                $definition->addArgument($handler['ident']);
            }
```